### PR TITLE
Log warning when no operations are found and add number of operations matched during generation

### PR DESCRIPTION
**Pull request by jerelmiller**
When generating a persisted query manifest file, its possible that no operations were found during manifest generation, which could be due to a misconfiguration of the glob pattern used to search for operations. Currently the user has to run the command, then open the manifest file to see if operations were added which can be pretty cumbersome. This PR updates the terminal output to list the number of matched operations. If none were found, a warning is displayed as well.

### CODE REVIEW

File: packages/generate-persisted-query-manifest/cli.js [hunk 1]
```diff
@@ -65,7 +66,20 @@ program
     );
     writeFileSync(outputPath, JSON.stringify(manifest, null, 2));
 
-    console.log(`Manifest written to ${outputPath}`);
+    if (manifest.operations.length === 0) {
```
Comment (jerelmiller) [inline packages/generate-persisted-query-manifest/cli.js:69]: I opted for a hybrid approach by logging a warning in addition to showing the number of matched operations in the log below. This gives us a chance to educate the user on where to find more information on configuring the glob pattern.

Here is what it looks like in the terminal:
Screenshot 2023-08-23 at 5 29 24 PM

Feedback welcome if that warning is too distracting or too wordy.

File: packages/generate-persisted-query-manifest/cli.js [hunk 2]
```diff
@@ -65,7 +66,22 @@ program
...
+            "files in this project. See the README for more information on how to configure the glob pattern: " +
+            "https://www.npmjs.com/package/@apollo/generate-persisted-query-manifest\n",
+        ),
+      );
+    }
+
+    console.log(
+      `${chalk.green("✓")} Manifest written to ${outputPath} with ${
+        manifest.operations.length
+      } operations.`,
```
Comment (glasser) [inline packages/generate-persisted-query-manifest/cli.js:83]: Special case 1?

Comment (jerelmiller) [reply packages/generate-persisted-query-manifest/cli.js:83]: 🤦‍♂️ n00b moment. Thanks!

Comment (jerelmiller) [reply packages/generate-persisted-query-manifest/cli.js]: Fixed in [d5b9bb6](https://github.com/apollographql/apollo-utils/pull/348/commits/d5b9bb6ff051627e514e95fa9c4bd3527903748c)

**Review by glasser — APPROVED** [review]
I think this was the last thing on your part of the board — merge https://github.com/apollographql/apollo-utils/pull/346 when this is merged?

**Comment by jerelmiller** [comment]
> merge https://github.com/apollographql/apollo-utils/pull/346 when this is merged?

Yes!
